09_maps/main.go

```diff
@@ -94,35 +111,76 @@ func main() {
 	// - "tall" -> 1 if height > 200, 0 otherwise
```

Comment:
@nobe4 Does this task also need adjustment for mix of types, or do I have to find a way?

Reply:
`1` and `0` are numbers, so you can use `int` as for the other values.